src/signature.cc

```diff
@@ -92,5 +94,17 @@ string GenerateCheckRequestSignature(const CheckRequest& request) {
   return hasher.Digest();
 }
 
+string GenerateAllocateQuotaRequestSignature(
+    const AllocateQuotaRequest& request) {
+  MD5 hasher;
+  const QuotaOperation& operation = request.allocate_operation();
+  hasher.Update(operation.method_name());
+
+  hasher.Update(kDelimiter, kDelimiterLength);
+  hasher.Update(operation.consumer_id());
```

Comment:
For our current ESP,  just method name is good enough.  But I think it is safer to add metric names for the future.